src/main/java/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,11 +1,19 @@
 package bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
 public record CreateBookRequestDto(
+        @NotBlank
         String title,
+        @NotBlank
         String author,
+        @NotBlank
         String isbn,
+        @Min(0)
```

Comment:
can you really have a book, which costs nothing?)

if yes - `@PositiveOrZero`

if no - `@Positive`